zanata-war/src/main/java/org/zanata/action/VersionGroupJoinAction.java

```diff
@@ -149,14 +152,34 @@ public String send() {
         }
     }
 
-    @AllArgsConstructor
-    public final class SelectableProject {
+    public void resetData() {
```

Comment:
What is the point of this? Please add documentation on this, and add a comment where it is used that indicates why it is used there.